# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mini_sendmail/hotfix-install.patch
# Copyright (C) 2007 - 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- mini_sendmail-1.3.9/Makefile.vanilla	2022-03-31 23:55:11.986903727 +0200
+++ mini_sendmail-1.3.9/Makefile	2022-03-31 23:55:31.374904727 +0200
@@ -4,8 +4,8 @@
 # Solaris, you will need to uncomment this definition.
 #SYSV_LIBS =    -lnsl -lsocket
 
-BINDIR =	/usr/local/sbin
-MANDIR =	/usr/local/man
+BINDIR =	$(prefix)/sbin
+MANDIR =	$(prefix)/man
 CC =		cc
 CFLAGS =	-O -ansi -pedantic -U__STRICT_ANSI__ -Wall -Wpointer-arith -Wshadow -Wcast-qual -Wcast-align -Wstrict-prototypes -Wmissing-prototypes -Wmissing-declarations -Wredundant-decls -Wno-long-long
 LDFLAGS =	-s -static
@@ -30,10 +32,10 @@
 
 
 install:	all
-	rm -f $(BINDIR)/mini_sendmail
-	cp mini_sendmail $(BINDIR)
-	rm -f $(MANDIR)/man8/mini_sendmail.8
-	cp mini_sendmail.8 $(MANDIR)/man8
+	rm -f $(DESTDIR)$(BINDIR)/mini_sendmail
+	cp mini_sendmail $(DESTDIR)$(BINDIR)
+	rm -f $(DESTDIR)$(MANDIR)/man8/mini_sendmail.8
+	cp mini_sendmail.8 $(DESTDIR)$(MANDIR)/man8
 
 clean:
 	rm -f mini_sendmail *.o core core.* *.core
